features/attribution-reporting.yml

```diff
@@ -1,12 +1,13 @@
-draft_date: 2024-10-28
-name: Attribution Reporting
-description: TODO
+name: Attribution reporting
+description: The attribution reporting API measures and attributes conversions to advertisements a user interacted with on another site.
```

Comment:
```suggestion
description: The Attribution Reporting API measures when an ad click or view leads to a conversion, such as a purchase on an advertiser site.
```

Reply:
I like this one, lifted from https://developers.google.com/privacy-sandbox/private-advertising/attribution-reporting